src/webview/generateInboundEventEditSequence.js

```diff
@@ -93,6 +93,8 @@ function doElementsDifferInterestingly(
             && oldBackgroundData.flags[flagName][oldElement.message.id]
               !== newBackgroundData.flags[flagName][newElement.message.id],
         )
+        || oldBackgroundData.mutedUsers.get(oldElement.message.sender_id)
+          !== newBackgroundData.mutedUsers.get(newElement.message.sender_id)
```

Comment:
Ha, right! We'll hope to switch to message-list diffing soon, right. I could put in a fix in this deprecated live code—basically check for any change at all to the muted-users state—but maybe not, in case I introduce some regression while trying to do so?

Reply:
> > Hmm, yeah, good catch!
> 
> You get the credit for this one: you asked for the test case that caught the bug! :stuck_out_tongue:

:-)

> > In fact -- does this work correctly in the live code today? It looks to me like it probably doesn't
> 
> Ha, right! We'll hope to switch to message-list diffing soon, right.
> 
> I could put in a fix in this deprecated live code—basically check for any change at all to the muted-users state—but maybe not, in case I introduce some regression while trying to do so?

Yeah. For this PR I think just adding a TODO comment in the live code would be fine.